src/main/kotlin/org/malachite/estella/queues/RabbitMqConsumers.kt

```diff
@@ -1,25 +1,48 @@
 package org.malachite.estella.queues
 
 import com.rabbitmq.client.Channel
+import org.malachite.estella.queues.utils.MsgDeserializer
+import org.malachite.estella.services.TaskService
+import org.malachite.estella.services.TaskStageService
 import org.springframework.amqp.rabbit.annotation.RabbitListener
 import org.springframework.amqp.support.AmqpHeaders
+import org.springframework.beans.factory.annotation.Autowired
 import org.springframework.messaging.handler.annotation.Header
 
-class RabbitMqConsumers {
+class RabbitMqConsumers() {
+
+    @Autowired private lateinit var taskService: TaskService
+    @Autowired private lateinit var taskStageService: TaskStageService
```

Comment:
Przeniósłbym do konstruktora klasy. Wtedy na pewno to będziesz miał i nie będzie to varem. Wiem, że kotlin to nie język funkcyjny, ale tu też się staramy unikać varów, jak tylko możemy xDD

Reply:
Plus jeśli to będzie w konstruktorze, to będzie to łatwiej ewentualnie przetestować unitowo (wiem, nie ten case), podając mocki bezpośrednio, a nie bawiąc się w mock beany.

Reply:
a, to jeszcze lepiej. Przyjmij odpowiednie serwisy jako parametry tej funkcji i przekaż je jako argumenty klasy. Easy as that.